CHANGELOG.md

```diff
@@ -6,7 +6,10 @@
 
 ##### Enhancements
 
-* None.
+* Added a config option (--undocumented-text) to set the default text for
+  undocumented code.
```

Comment:
Per CONTRIBUTING.md, need two spaces after the end of the description please :)

Reply:
whoops, sorry about that